src/verinfast/config.py

```diff
@@ -229,6 +229,13 @@ def __init__(self, cfg_path: str = None) -> None:
             """
             self.modules = ConfigModules(code=cm, cloud=[])
             self.runGit = False
+        elif (
+            "repos" in self.config and
+            "modules" in self.config and
```

Comment:
what about local repos?

Reply:
("repos" in self.config or "local_repos" in self.config)

Reply:
shouldn't run git on local repos we don't clone those or change branches (since we can't change back)

Reply:
We lose a lot of widgets then if we run on local repo, so we shouldn't suggest it to targets.